pkg/alertmanager/multitenant.go

```diff
@@ -226,12 +233,15 @@ func newMultitenantAlertmanagerMetrics(reg prometheus.Registerer) *multitenantAl
 // organizations.
 type MultitenantAlertmanager struct {
 	services.Service
+	alertmanagerpb.AlertmanagerClient
+	grpc_health_v1.HealthClient
```

Comment:
Why the `MultitenantAlertmanager` needs to implement `grpc_health_v1.HealthClient`?